Apps/Src/RTOSPedals.c

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "RTOSPedals.h"
+
+/**
+* @brief   Initialize both pedals to 0%
+* @param   None
+* @return  None
+*/ 
+static void Pedals_Values_Init(void){
+    accelerator_percentage = 0;
+    brake_percentage = 0;
+}
+
+/**
+* @brief   Read both pedal percentage values and set the global variables
+* @param   None
+* @return  None
+*/ 
+void Task_ReadPedals(void *p_arg){
+    Pedals_Values_Init();
+    
+    OS_ERR err;
+
+    while(1){
```

Comment:
Add a delay in this loop. Doesn't have to be long, but we don't need to be wasting all our cycles on reading from pedals.